NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
```

Comment:
This was needed to pull in some of the testing related packages for the test project. The new versions are not on NuGet.org.